src/components/Editors.jsx

```diff
@@ -76,7 +77,8 @@ class Editors extends React.Component {
                         height="auto"
                         onChange={this.handleChange('onHTMLChange')}
                         showPrintMargin={false}
-                        editorProps={{$blockScrolling: Infinity}}
+                        editorProps={{$blockScrolling: Infinity, enableEmmet: true}}
+                        setOptions={{enableEmmet: true}}
```

Comment:
`enableEmmet` option on this line is unnecessary. It's my mistake.

Do you mean add a option on `Actions` menu, or let `setOptions` as a prop of Editors component?